tests/testthat/test-proj.R

```diff
@@ -48,6 +48,14 @@ test_that("proj_path() appends to the project path", {
   expect_identical(proj_path("a", "b", "c"), proj_path("a/b/c"))
 })
 
+test_that("proj_path() errors with absolute paths", {
+  create_local_project()
+  expect_error(
```

Comment:
I think we favor snapshot tests over `expect_error(regexp =)` these days.

I'll leave it up to you whether to change.